[2024-10-05T13:29:53.852-0400] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-05T13:29:57.616-0400] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: huggingface_pdf_pipeline.upload_pdfs_task scheduled__2024-10-04T00:00:00+00:00 [queued]>
[2024-10-05T13:29:57.902-0400] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: huggingface_pdf_pipeline.upload_pdfs_task scheduled__2024-10-04T00:00:00+00:00 [queued]>
[2024-10-05T13:29:57.905-0400] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-05T13:29:58.118-0400] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): upload_pdfs_task> on 2024-10-04 00:00:00+00:00
[2024-10-05T13:29:58.124-0400] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'huggingface_pdf_pipeline', 'upload_pdfs_task', 'scheduled__2024-10-04T00:00:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/huggingface_pdf_airflow_dag.py', '--cfg-path', '/tmp/tmpjc4q4qcg']
[2024-10-05T13:29:58.122-0400] {standard_task_runner.py:72} INFO - Started process 1383 to run task
[2024-10-05T13:29:58.132-0400] {standard_task_runner.py:105} INFO - Job 58: Subtask upload_pdfs_task
[2024-10-05T13:30:21.921-0400] {job.py:229} INFO - Heartbeat recovered after 38.06 seconds
[2024-10-05T13:30:34.935-0400] {task_command.py:467} INFO - Running <TaskInstance: huggingface_pdf_pipeline.upload_pdfs_task scheduled__2024-10-04T00:00:00+00:00 [running]> on host LAPTOP-GD5THE3N.
[2024-10-05T13:31:11.829-0400] {job.py:229} INFO - Heartbeat recovered after 50.00 seconds
[2024-10-05T13:31:37.674-0400] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='huggingface_pdf_pipeline' AIRFLOW_CTX_TASK_ID='upload_pdfs_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-04T00:00:00+00:00'
[2024-10-05T13:31:37.675-0400] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-05T13:31:37.677-0400] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-05T13:31:37.677-0400] {logging_mixin.py:190} INFO - Current task name:upload_pdfs_task state:running start_date:2024-10-05 17:29:57.622767+00:00
[2024-10-05T13:31:37.678-0400] {logging_mixin.py:190} INFO - Dag name:huggingface_pdf_pipeline and current dag run status:running
[2024-10-05T13:31:37.679-0400] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-05T13:31:37.681-0400] {logging_mixin.py:190} INFO - Error uploading / to Azure Blob Storage Container 1: Please specify a container name and blob name.
[2024-10-05T13:31:37.683-0400] {logging_mixin.py:190} INFO - Error uploading h to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'h'
[2024-10-05T13:31:37.684-0400] {logging_mixin.py:190} INFO - Error uploading o to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'o'
[2024-10-05T13:31:37.685-0400] {logging_mixin.py:190} INFO - Error uploading m to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'm'
[2024-10-05T13:31:37.687-0400] {logging_mixin.py:190} INFO - Error uploading e to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'e'
[2024-10-05T13:31:37.688-0400] {logging_mixin.py:190} INFO - Error uploading / to Azure Blob Storage Container 1: Please specify a container name and blob name.
[2024-10-05T13:31:37.689-0400] {logging_mixin.py:190} INFO - Error uploading p to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'p'
[2024-10-05T13:31:37.690-0400] {logging_mixin.py:190} INFO - Error uploading a to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'a'
[2024-10-05T13:31:37.690-0400] {logging_mixin.py:190} INFO - Error uploading t to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 't'
[2024-10-05T13:31:37.691-0400] {logging_mixin.py:190} INFO - Error uploading r to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'r'
[2024-10-05T13:31:37.691-0400] {logging_mixin.py:190} INFO - Error uploading i to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'i'
[2024-10-05T13:31:37.692-0400] {logging_mixin.py:190} INFO - Error uploading c to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'c'
[2024-10-05T13:31:37.693-0400] {logging_mixin.py:190} INFO - Error uploading k to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'k'
[2024-10-05T13:31:37.694-0400] {logging_mixin.py:190} INFO - Error uploading / to Azure Blob Storage Container 1: Please specify a container name and blob name.
[2024-10-05T13:31:37.696-0400] {logging_mixin.py:190} INFO - Error uploading D to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'D'
[2024-10-05T13:31:37.705-0400] {logging_mixin.py:190} INFO - Error uploading o to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'o'
[2024-10-05T13:31:37.707-0400] {logging_mixin.py:190} INFO - Error uploading w to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'w'
[2024-10-05T13:31:37.709-0400] {logging_mixin.py:190} INFO - Error uploading n to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'n'
[2024-10-05T13:31:37.710-0400] {logging_mixin.py:190} INFO - Error uploading l to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'l'
[2024-10-05T13:31:37.712-0400] {logging_mixin.py:190} INFO - Error uploading o to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'o'
[2024-10-05T13:31:37.714-0400] {logging_mixin.py:190} INFO - Error uploading a to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'a'
[2024-10-05T13:31:37.715-0400] {logging_mixin.py:190} INFO - Error uploading d to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'd'
[2024-10-05T13:31:37.716-0400] {logging_mixin.py:190} INFO - Error uploading s to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 's'
[2024-10-05T13:31:37.717-0400] {python.py:240} INFO - Done. Returned value was: None
[2024-10-05T13:31:47.536-0400] {job.py:229} INFO - Heartbeat recovered after 36.27 seconds
[2024-10-05T13:31:52.491-0400] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-05T13:31:52.493-0400] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=huggingface_pdf_pipeline, task_id=upload_pdfs_task, run_id=scheduled__2024-10-04T00:00:00+00:00, execution_date=20241004T000000, start_date=20241005T172957, end_date=20241005T173152
[2024-10-05T13:31:53.260-0400] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-05T13:31:53.261-0400] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-05T13:31:53.262-0400] {logging_mixin.py:190} INFO - Dag name:huggingface_pdf_pipeline queued_at:2024-10-05 17:27:34.552395+00:00
[2024-10-05T13:31:53.262-0400] {logging_mixin.py:190} INFO - Task hostname:LAPTOP-GD5THE3N. operator:PythonOperator
[2024-10-05T13:31:57.770-0400] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-05T13:31:58.261-0400] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-10-05T13:32:15.922-0400] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-05T13:32:15.970-0400] {local_task_job_runner.py:245} INFO - ::endgroup::
